### PR TITLE
JNG-5144 replace logger

### PR DESCRIPTION
**Pull request by robertcsakany**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-5144" title="JNG-5144" target="_blank">TaskJNG-5144</a>  Replace hacky own logger to standard SLF4J API logger
  </td></table>
  <br />

### CODE REVIEW

File: src/main/java/hu/blackbelt/epsilon/maven/plugin/MavenLog.java [hunk 1]
```diff
@@ -108,11 +109,297 @@ public static LogLevel determinateLogLevel(org.apache.maven.plugin.logging.Log l
...
+    public void trace(Marker marker, String format, Object arg1, Object arg2) {
+    }
+
+    @Override
+    public void trace(Marker marker, String format, Object... argArray) {
+    }
+
+    @Override
+    public void trace(Marker marker, String msg, Throwable t) {
+    }
```
Comment (bencegelei) [inline src/main/java/hu/blackbelt/epsilon/maven/plugin/MavenLog.java:164]: Why are these functions empty?

File: src/main/java/hu/blackbelt/epsilon/maven/plugin/execute/ExecuteEpsilonMojo.java [hunk 2]
```diff
@@ -90,15 +90,15 @@ synchronized public void execute() throws MojoExecutionException, MojoFailureExc
...
+        Logger log = new MavenLog(getLog());
 
         Map<String, Object> injectedContextMap = Maps.newHashMap();
         if (injectedContexts != null) {
             for (InjectedContext i : injectedContexts) {
                 try {
                     injectedContextMap.put(i.getName(), forName(i.getClazz()).newInstance());
                 } catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
-                    log.warn(e);
+                    log.warn("Error", e);
```
Comment (bencegelei) [inline src/main/java/hu/blackbelt/epsilon/maven/plugin/execute/ExecuteEpsilonMojo.java:101]: :D

File: src/main/java/hu/blackbelt/epsilon/maven/plugin/MavenLog.java (hunk 1 above)
Comment (robertcsakany) [reply src/main/java/hu/blackbelt/epsilon/maven/plugin/MavenLog.java:164]: Trace is not supported by maven.

File: src/main/java/hu/blackbelt/epsilon/maven/plugin/execute/ExecuteEpsilonMojo.java (hunk 2 above)
Comment (robertcsakany) [reply src/main/java/hu/blackbelt/epsilon/maven/plugin/execute/ExecuteEpsilonMojo.java:101]: This is it :)

**Review by bencegelei — APPROVED** [review]

**Pull request merged**